asset/CMakeLists.txt

```diff
@@ -5,6 +5,15 @@ if(APPLE)
 	SET(ASSETS ${ASSETS}
 		${CMAKE_CURRENT_SOURCE_DIR}/brd.icns
 	)
+elseif(UNIX)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/ofbv-app.svg
+		DESTINATION share/icons/hicolor/scalable/apps RENAME openboardview.svg)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.appdata.xml
+		DESTINATION share/appdata)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.desktop
+		DESTINATION share/applications)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.xml
+		DESTINATION share/mime/packages)
```

Comment:
I think the paths should be `/usr/share/` not `share/`. CMake will prefix relative paths with `CMAKE_INSTALL_PREFIX` (usually `/usr/local` or `/usr`), but these files should always go under `/usr`.